dart/lib/src/protocol/sdk_version.dart

```diff
@@ -41,8 +41,9 @@ class SdkVersion {
     List<String> integrations,
     List<SentryPackage> packages,
   })  : assert(name != null || version != null),
-        _integrations = integrations ?? [],
-        _packages = packages ?? [];
+        // List.from prevents from having immutable lists
+        _integrations = integrations != null ? List.from(integrations) : [],
+        _packages = packages != null ? List.from(packages) : [];
```

Comment:
now you can fix `SentryFlutter._setSdk` as I needed to do `List.from` manually in there, now the ctor does it automatically